common/src/main/java/com/wynntils/overlays/minimap/MinimapOverlay.java

```diff
@@ -242,54 +246,47 @@ private void renderPois(
             cosRotationRadians = (float) -StrictMath.cos(rotationRadians);
         } else {
             sinRotationRadians = 0f;
-            cosRotationRadians = 0f;
+            cosRotationRadians = 1f;
```

Comment:
This is the "sane" default value, these rotations applied to the render coordinates produce the same output as the original input.